src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
+type Color = 'red' | 'green' | 'blue';
```

Comment:
But better would be to use `enum` here 🙃
In this case, you will be able to do ` shape: Shape = Shape.triangle;` instead of ` shape: Shape = 'triangle';`
You will avoid using "magic value"